docs/sources/installation/helm/install-scalable/index.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Install the Helm Chart
+menuTitle: Install Loki
+description: Install Loki in scalable mode.
+aliases:
+  - /docs/writers-toolkit/latest/templates/task-template
+weight: 100
+keywords:
+  - keyword
+  - key
+  - word
+---
+
+# Install the Helm Chart
+<!-- vale Grafana.Quotes = NO -->
+<!-- vale Grafana.Quotes = YES -->
+
+This Helm Chart installation runs the Grafana Loki cluster within a Kubernetes cluster.
+
+By default, this chart configures Loki to run `read` and `write` targets in a [scalable](../../fundamentals/architecture/deployment-modes/#simple-scalable-deployment-mode), highly available architecture (3 replicas of each) designed to work with AWS S3 object storage. It will also configure meta-monitoring of metrics and logs and use object store
+
+**Before you begin:**
+
+- Helm 3 or above. See [Installing Helm](https://helm.sh/docs/intro/install/).
+- A running Kubernetes cluster.
+
+**To deploy Loki in scalable mode:**
+
+1. Add [Grafana's chart repository](https://github.com/grafana/helm-charts) to Helm:
+
+    ```bash
+    helm repo add grafana https://grafana.github.io/helm-charts
+    ```
+
+1. Update the chart repository:
+
+    ```bash
+    helm repo update
+    ```
+
+1. Configure the object storage:
+
+    - Create the configuration file `values.yaml`:
+
+        ```yaml
+        storage:
+            bucketNames:
+            chunks: chunks
+            ruler: ruler
+            admin: admin
+            type: s3
+            s3:
+            s3: null
+            endpoint: null
+            region: null
+            secretAccessKey: null
+            accessKeyId: null
+            s3ForcePathStyle: false
+            insecure: false
+        ```
+
+    - Define the AWS S3 credentials in the file.
```

Comment:
@trevorwhitney how would on inject environment variables instead to manage secrets?

Reply:
I imagine you would just use ENV_VARS in your values.yaml, and then set `--config.expand-env`. Not sure if we make that easy though?

Reply:
You mean `--config.expand-env` from Loki? Hm, we should always start Loki with `--config.expand-env` then.

Reply:
Yup, from loki. I don't agree we should always start loki that way as there are security reasons that's not enabled by default. However, we should have any easy way to toggle it on from the helm chart.